static/app/views/starfish/components/fullSpanDescription.tsx

```diff
@@ -8,14 +8,15 @@ import {SQLishFormatter} from 'sentry/views/starfish/utils/sqlish/SQLishFormatte
 
 const formatter = new SQLishFormatter();
 
-export function FullQueryDescription({group, shortDescription}: Props) {
+export function FullSpanDescription({group, shortDescription, language = 'sql'}: Props) {
```

Comment:
m: I'd rather not make sql the default - no formatting should be the default. This also means we can remove the `language="http"` prop when it isn't being used.

By making that change, we can also make `language` optional, and change the type in props interface to be `language?: sql`.

Reply:
Made the change!